docs/references/strategies/languages/maven/maven.md

```diff
@@ -14,7 +14,15 @@ Maven analysis attempts these analysis methods in order:
 3. Run the maven plugin command version 3.3.0.
 4. Scan `pom.xml` files located in the file tree.
 
+FOSSA CLI uses [strategy command selection](../../../../features/strategy-command-selection.md) for Maven commands:
+- Maven _candidate commands_ are:
+  - If present, the command specified by the `FOSSA_MAVEN_CMD` environment variable.
+  - If present, the `mvnw` or `mvnw.bat` script in the project directory or any ancestor directory.
+  - Finally, the `mvn` command, which is searched for in `$PATH`.
+- The Maven _evaluation flag_ is `-v` for each _candidate command_.
```

Comment:
I wouldn't change anything here, but I would add something like the following as a TLDR so user's don't have to read through the `strategy command selection` doc to understand what a `evaluation flag` is.
```
The FOSSA CLI tries `./mvnw -v` before moving onto `mvn -v` and selects the first one that succeeds to run the analysis tactic.
```